src/clarity-angular/popover/common/popover.ts

```diff
@@ -45,7 +45,7 @@ export class Popover {
 
     constructor(private element: any) {
         // Browsers don't agree with what to do if some of these are not specified, so we set them all to be safe.
-        element.style.position = POSITION_ABSOLUTE;
+        element.style.position = POSITION_FIXED;
```

Comment:
Wait, when the popover is open it needs `position: absolute`. I was expecting the `position: fixed` to be used only when the popover is closed (which you already do with the CSS class, so I don't see why you needed to change it here too).

Reply:
👍